src/main/java/com/theelixrlabs/healthcare/constants/DoctorConstants.java

```diff
@@ -8,8 +8,11 @@ public class DoctorConstants {
     public static final String CHARACTER_ONLY_REGEX_PATTERN = "^[a-zA-Z]+$";
     public static final String COMMA_DELIMITER = ", ";
     public static final String CREATE_DOCTOR_END_POINT = "/saveDoctors";
+    public static final String DOCTOR_NAME_PARAM = "name";
     public static final String DOCTORS_COLLECTION_NAME = "doctors";
     public static final String EMPTY_SPACE = " ";
     public static final String GET_DOCTOR_BY_ID_ENDPOINT = "/getDoctorById/{doctorId}";
```

Comment:
Rename it to "/doctors/{doctorId}", Http method will show its GET/POST/PATCH

Reply:
changed it into "/doctors/{doctorId}"